commit 8ca072f295023b2e9bf786913770ca9636379514
Author: Magnus Lilja <lilja.magnus@gmail.com>
Date:   Wed Dec 21 22:13:58 2016 +0100

    dmaengine: ipu: Make sure the interrupt routine checks all interrupts.
    
    commit adee40b265d7568296e218f079f478197ffa15bf upstream.
    
    Commit 3d8cc00073d6 ("dmaengine: ipu: Consolidate duplicated irq handlers")
    consolidated the two interrupts routines into one, but the remaining
    interrupt routine only checks the status of the error interrupts, not the
    normal interrupts.
    
    This patch fixes that problem (tested on i.MX31 PDK board).
    
    Fixes: 3d8cc00073d6 ("dmaengine: ipu: Consolidate duplicated irq handlers")
    Cc: Vinod Koul <vinod.koul@intel.com>
    Signed-off-by: Magnus Lilja <lilja.magnus@gmail.com>
    Signed-off-by: Vinod Koul <vinod.koul@intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1348_8ca072f295023b2e9bf786913770ca9636379514.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1348_8ca072f295023b2e9bf786913770ca9636379514.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
